scripts/ci/analyze-commits.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+/*
+ * This script analyzes the current commits of the CI.
+ * It will search for blocked statements, which have been added in the commits and fail if present.
+ */
+
+var child_process = require('child_process');
+var blocked_statements = [
+  'ddescribe',
+  'fdescribe',
+  'iit',
+  'fit',
+  'xdescribe',
+  'xit',
+  'debugger;'
+];
+
+var blockedRegex = new RegExp('\\+.*(' + blocked_statements.join('|') + ').*$', 'mg');
+
+var diff = child_process.execSync('git diff --unified=0 HEAD~1').toString();
+var isInvalid = blockedRegex.test(diff);
+
+if (isInvalid) {
+  console.warn('Warning: You are using a statement in your commit, which is not allowed.\n' +
```

Comment:
`console.warn` is an alias for `console.error`

See the node reference: https://nodejs.org/api/console.html#console_console_warn_data

But if you want me to change it, let me know.